client/src/app/shared/video/video-miniature.component.html

```diff
@@ -14,12 +14,81 @@
     </a>
 
     <span i18n class="video-miniature-created-at-views">{{ video.publishedAt | myFromNow }} - {{ video.views | myNumberFormatter }} views</span>
+  
+    <div class="dropdown" ngbDropdown placement="bottom-right" role="group">
+
+      <button class="icon-action" aria-label="Actions on this video" title="Actions on this video" 
+              id="moreOptions" data-flip="true"
+              (click)="loadDropdownInformations()"
```

Comment:
Maybe `mouseover`?

Reply:
It's an action button, not an info button.

Reply:
The idea is to have the information loaded before the user clicks on the button

Reply:
Ah, I see, sorry. I thought you meant to open the dropdown on mouseover.

Reply:
@Chocobozzz working like a charm!